cliquet/__init__.py

```diff
@@ -79,6 +79,14 @@
 }
 
 
+class Service(CorniceService):
+    """Subclass of the default cornice service.
+
+    This is useful in order to attach specific behaviours without monkey
+    patching the default cornice service (which would impact other uses of it)
+    """
```

Comment:
We are extending `CorniceService` and reusing it in all non-resource based views.